src/coreclr/inc/CrstTypes.def

```diff
@@ -201,6 +201,10 @@ End
 Crst Exception
 End
 
+Crst ExecutableAllocatorLock
+    AcquiredAfter LoaderHeap LeafLock
```

Comment:
No lock should be nested inside LeafLock.

This means that some uses of LeafLock are not valid anymore, and should get its own level.